xmodule/js/spec/video/video_caption_spec.js

```diff
@@ -1327,5 +1331,48 @@
                     });
                 });
         });
+
+        describe('google disclaimer', () => { 
+            const BASE_CAPTIONS = [
+                'this is a caption',
+                'this is another caption',
+                'quite lovely weather we\'re having today',
+                'indeed',
+            ]
+            var Caption;
+
+            const makeCaptions = (provider) => {
+                const providerAttr = provider ? ` data-provider="${provider}"` : '';
+                const aiGeneratedTag = `<span id="captions-ai-generated" ${providerAttr}></span>`;
+                var captions = [...BASE_CAPTIONS];
```

Comment:
```suggestion
                const captions = [...BASE_CAPTIONS];
```
This should be fine as a `const`